tests/ui/resolve/transient.sol

```diff
@@ -0,0 +1,3 @@
+contract A {
+    public transient tempValue; //~ ERROR: expected contract item (function, variable, struct, or modifier definition), found keyword `public`
```

Comment:
missing type, this is the wrong error

Reply:
changed the test and removed the err, with the assertion the test would actually panic